extensions/dig/src/index.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  Detail,
+  Icon,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  useNavigation,
```

Comment:
Switched to the correct ones, some were deprecated 😄 

```suggestion
  ActionPanel,
  Detail,
  Icon,
  List,
  Action,
  showToast,
  Toast,
  useNavigation,
```